react-challenge.html

```diff
@@ -34,13 +34,15 @@
         }
 
         handleMoodOptions = () => {
-          let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          const { moods } = this.state 
```

Comment:
good use of Destructuring, did this work out for you?

Reply:
yes overall it worked i had to write it out differently but it gave me the same result